template/tinyvue/package.json

```
@@ -112,6 +112,7 @@
     "tsx": "^4.19.1",
     "typescript": "^4.9.5",
```

Biome (1.9.4)

[error] The key typescript was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)